newrelic/exporter.go

```diff
@@ -133,24 +133,25 @@ func NewExportPipeline(service string, traceOpt []sdktrace.TracerProviderOption,
 
 // InstallNewPipeline installs a New Relic exporter with default settings
 // in the global OpenTelemetry telemetry pipeline. It is the callers
-// responsibility to stop the returned push Controller. This function uses the
-// following environment variables to configure the exporter installed in the
-// pipeline:
-//
-//    * `NEW_RELIC_API_KEY`: New Relic Event API key.
+// responsibility to stop the returned push Controller. 
+// ## Prerequisites
+// To use this, make sure you have the following:
```

Comment:
Rather than duplicate this how-to information, It would probably be better to direct the reader to the 
"Getting Started" section in the README.md file. This will eliminate redundancy and keep this from becoming out of date.

Reply:
...and by "this how-to information," I mean the two lines about New Relic account and New Relic Insights insert key.

Reply:
Rich, I took your suggestion and inserted that link with an anchor to the exact section. See if this looks better.